docs/INSTALLATION.md

```diff
@@ -1 +1 @@
-Methods for installing Rust change as the language evolves. To get up-to-date installation instructions head to the [Rust homepage](https://www.rust-lang.org/) and click the "Install Rust" link.
+Methods for installing Rust change as the language evolves. To get up-to-date installation instructions head to the [following page](https://www.rust-lang.org/tools/install).
```

Comment:
I prefer a text other than "following page" for reasons listed in "Links should make sense out of context." section at https://webaim.org/techniques/hypertext/ . Perhaps something like "Rust install page"